frontend/src/components/Movie.jsx

```diff
@@ -1,23 +1,28 @@
 import React from "react";
 import PropTypes from "prop-types";
-
+// fonction d'affichage du titre, image et description du film
 function Movie({ movie }) {
   return (
-    <section>
-      <h1>{movie.title}</h1>
-      <img
-        src={`https://image.tmdb.org/t/p/w500${movie.backdrop_path}`}
-        alt={movie.path}
-      />
-    </section>
+    <div>
+      <h5>{movie.title}</h5>
```

Comment:
div should not have titles : section should

Reply:
done